render/flowmill.render

```diff
@@ -241,13 +241,7 @@ app agent_internal {
       5: u8 stream_type       // enum STREAM_TYPE { ST_SEND = 0, ST_RECV = 1 }
       6: u8 client_server     // enum CLIENT_SERVER_TYPE { SC_CLIENT = 0, SC_SERVER = 1 }
     }
-    25: log nic_queue_state {
```

Comment:
We don't have to leave a 'gap' here to hold a position for older messages?  Or is the assumption that we're just pushing stuff 'afresh,' and so won't have older messages to deal with?

Reply:
These messages are internal to the kernel collector (sent from ebpf code to userspace), and are thus not part of some public interface that must be kept compatible with older versions.